lib/modules/manager/gomod/artifacts.ts

```diff
@@ -281,13 +284,29 @@ export async function updateArtifacts({
       execCommands.push(`${cmd} ${args}`);
     }
 
+    const goWorkSumFileName = upath.join(goModDir, 'go.work.sum');
     if (useVendor) {
-      args = 'mod vendor';
-      logger.debug('go mod tidy command included');
-      execCommands.push(`${cmd} ${args}`);
-      if (isGoModTidyRequired) {
-        args = 'mod tidy' + tidyOpts;
```

Comment:
you completely lost this part. is that expected? 🤔

Reply:
Yes, this was removed in my first commit, intentionally as it's redundant. There's a `goModTidy`  above that should handle this since the condition above is also on `isGoModTidyRequired` so it was just running `go mod tidy` twice in a row when it only needs to run once.

Reply:
This was intentional actually. Somehow the tidying wasn't complete after one so we intentionally ran it twice. In theory it shouldn't be needed

Reply:
So I thought the same when I started on this, but when I looked at the commits and comments, it looks like what you're talking about is running `go mod tidy` _after_ vendoring (03b3f09d37). I think once before, once after should be sufficient for all cases.

Reply:
I just added it back to simplify review.